code/src/components/InputForm.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react'
+
+const InputForm = () =>{
+    const [message, setMessage] = useState("");
+    const THOUGHTS_URL = "https://happy-thoughts-technigo.herokuapp.com/thoughts";
+    
+    const submit = (event) => {
+        event.preventDefault();
+
+        fetch(THOUGHTS_URL, {
+          method: "POST",
+          headers: { "Content-Type": "application/json" },
+          body: JSON.stringify({ message: message }),
```

Comment:
Since both key and value of this object have same name, we can simplify this : 

```
body: JSON.stringify({ message }),
```

Reply:
Updated  to body: JSON.stringify({ message }),